admin/components/Home/WithoutSiteId.js

```diff
@@ -32,7 +30,10 @@ const WithoutSiteId = ({
     <>
       <Container>
         <InnerContainer>
-          <Image alt="Frontity Theme" src={`${siteUrl}${frontityThemeImage}`} />
+          <Image
+            alt="Frontity Theme"
+            src={`${siteUrl}/wp-content/plugins/wp-pwa/admin/assets/frontity-theme.png`}
```

Comment:
This is going to generate problems when the exact folder of our plugin is not `wp-pwa`. We've found this problem in the past. For that reason we use a PHP global called `wp_pwa_path` in the injector:
https://github.com/frontity/wp-plugin/blob/dev/wp-pwa.php#L814

In your case it's not the path but the URL. But WP has a function called `plugin_dir_url`:
https://codex.wordpress.org/Function_Reference/plugin_dir_url
I think you should provide that value to MST so it can use it for things like this one.